src/redux/blockchain/blockchainActions.js

```diff
@@ -78,10 +80,14 @@ export const connect = () => {
           });
           ethereum.on("chainChanged", () => {
             window.location.reload();
-          });
+          })
           // Add listeners end
         } else {
-          dispatch(connectFailed(`Change network to ${CONFIG.NETWORK.NAME}.`));
+          //dispatch(connectFailed(`Change network to ${CONFIG.NETWORK.NAME}.`));
+          await window.ethereum.request({
+            method: 'wallet_switchEthereumChain',
+            params: [{chainId:CONFIG.NETWORK.HEX_ID}],
```

Comment:
Instead of adding to the `config.json` just leverage the web3 utils like this: `Web3.utils.toHex(CONFIG.NETWORK.ID)`

Reply:
I agree, this method generic more than my code.